api/control/v1alpha1/types.go

```diff
@@ -56,6 +56,24 @@ const TemplateExpansionAnnotationKey string = "control.kubestellar.io/expand-tem
 // WEC properties to be used in customization.
 const PropertyConfigMapNamespace = "customization-properties"
 
+// BindingPolicy defines in which ways the workload objects ('what') and the destinations ('where') are bound together.
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="SYNCED",type="string",JSONPath=".status.conditions[?(@.type=='Synced')].status"
+// +kubebuilder:printcolumn:name="READY",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
+// +kubebuilder:printcolumn:name="TYPE",type="string",JSONPath=".spec.type"
+// +kubebuilder:printcolumn:name="AGE",type="date",JSONPath=".metadata.creationTimestamp"
+// +kubebuilder:resource:scope=Cluster,shortName={bp}
+type BindingPolicy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   BindingPolicySpec   `json:"spec,omitempty"`
+	Status BindingPolicyStatus `json:"status,omitempty"`
+}
+
```

Comment:
I like the changes that are just re-ordering content in this file. I think it may be better to do them first as a distinct PR.

Reply:
If that is not a strong opinion, I'll keep them here.

Reply:
OK to leave them here.